website/docs/extensions/testing.md

```diff
@@ -19,15 +19,27 @@ The behaviour of the `scarb test` command can be changed by developers.
 To do so, provide a script named explicitly `test` in the current workspace `Scarb.toml`.
 If such script is found, Scarb will execute it instead of running the default test runner.
 
-For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry) for testing in
-your project, type the following:
+Scarb can be configured to use any tool in place of the default `cairo-test`, simply by changing
+this configuration:
 
 ```toml filename="Scarb.toml"
 [scripts]
-test = "snforge"
+test = "my-custom-command"
 ```
 
-Do not forget to properly set up Starknet Forge in your project beforehand.
+## Using Starknet Foundry
+
+[Starknet Foundry](https://foundry-rs.github.io/starknet-foundry) is another project developed by [Software Mansion](https://swmansion.com/) team.
```

Comment:
```suggestion
[Starknet Foundry](https://foundry-rs.github.io/starknet-foundry), like Scarb, is a project built by the [Software Mansion](https://swmansion.com/) team.
```

Reply:
I think `developed` sounds more profesional than `built`. We can do `like Scarb`, though I thought that it should be quite obvious to Scarb users who developed it.

Reply:
I am indifferent to build/developed but I would keep "like Scarb" to make is explicit. Do you know what companies created all of your cli tools? I am sure not, it is a bit of a knowledge bias.